src/reducers/errorsReducer.js

```diff
@@ -35,6 +36,8 @@ function errorsAllIds(state = [], { meta, type }) {
  * @param {object} [state=[]] - Current authError redux state
  * @param {object} action - Object containing the action that was dispatched
  * @param {string} action.type - Type of action that was dispatched
+ * @param action.meta
```

Comment:
Nit: Can we add types and descriptions to these - both should be object I believe